tests/test_query_parsing.py

```diff
@@ -40,174 +67,162 @@ async def test_nested_query(self):
     register_inference_services(aidb_engine, data_dir)
 
     config = aidb_engine._config
-
-    # test column alias, subquery
-    query_str1 = '''
-                 SELECT color as alias_color
-                 FROM colors02 LEFT JOIN objects00 ON colors02.frame = objects00.frame
-                 WHERE alias_color IN (SELECT table2.color AS alias_color2 FROM colors02 AS table2)
-                  OR colors02.object_id > (SELECT AVG(ob.object_id) FROM objects00 AS ob WHERE frame > 100);
-                 '''
-
-    normalized_query_str1 = ("SELECT colors02.color "
-                             "FROM colors02 LEFT JOIN objects00 ON colors02.frame = objects00.frame "
-                             "WHERE colors02.color IN (SELECT table2.color AS alias_color2 FROM colors02 AS table2) "
-                             "OR colors02.object_id > (SELECT AVG(ob.object_id) FROM objects00 AS ob WHERE frame > 100)")
-
-    # test replacing column with root column in filtering predicate,
-    # the root column of 'colors02.object_id' is 'objects00'
-    correct_fp = [['colors02.color IN (SELECT table2.color AS alias_color2 FROM colors02 AS table2)',
-                    'objects00.object_id > (SELECT AVG(ob.object_id) FROM objects00 AS ob WHERE frame > 100)']]
-
-    # filter predicates connected by OR are in same set
-    correct_service = [{'colors02', 'objects00'}]
-    correct_tables = [{'colors02', 'objects00'}]
-    self._test_query(query_str1, config, normalized_query_str1, correct_fp, correct_service, correct_tables, 3)
-
-
+    
+       
+    queries = {
+    "test_query_0": {
+        QueryParameters.QUERY_STR: '''
```

Comment:
Can you reformat here and below? Start with a new line with four space like
QueryParameters.QUERY_STR:
        '''
        SELECT
        FROM
        '''

Reply:
@ttt-77 Can you please let me know if I should reformat all the other fields (CORRECT_FP,etc) in this format as well, or just the query and normalized query fields?

Reply:
All

Reply:
Done, reformatted all the dictionaries.